services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Persistence.kt

```diff
@@ -92,17 +92,19 @@ enum class TodennakoinenHaittaPaaAjoRatojenKaistajarjestelyihin(
 }
 
 /** NOTE Järjestys täytyy olla pienimmästä suurimpaan */
-enum class KaistajarjestelynPituus(override val value: Int, override val explanation: String) :
-    Luokittelu {
-    YKSI(1, "Ei tarvita"),
-    KAKSI(2, "Enintään 10 m"),
-    KOLME(3, "11 - 100 m"),
-    NELJA(4, "101 - 500 m"),
-    VIISI(5, "Yli 500 m")
+enum class AutoliikenteenKaistavaikutustenPituus(
+    override val value: Int,
+    override val explanation: String
+) : Luokittelu {
+    YKSI(1, "Ei vaikuta kaistajärjestelyihin"),
```

Comment:
These explanations aren't used anywhere. Maybe we should delete them or move them to comments? Not necessarily in this PR, though.

Reply:
Removed explanations from all enums where they are not needed. Moved enums to Enums.kt files in root hanke package and tormaystarkastelu package.